lib/lambda_ethereum_consensus/application.ex

```diff
@@ -12,18 +12,25 @@ defmodule LambdaEthereumConsensus.Application do
     {args, _remaining_args, _errors} =
       OptionParser.parse(System.argv(), switches: [checkpoint_sync: :string])
 
-    {:ok, host} = Libp2p.host_new()
-    {:ok, gsub} = Libp2p.new_gossip_sub(host)
+    config = Application.get_env(:lambda_ethereum_consensus, :discovery)
+    port = Keyword.get(config, :port, 9000)
+    bootnodes = Keyword.get(config, :bootnodes, [])
```

Comment:
In general I think it's better to crash than to add default values. If we pass an empty string as bootnodes it's not going to work anyway and it's probably going to be harder to debug.

Reply:
If I understand correctly, bootnodes aren't required for peer discovery. They are just permanent peers.

Reply:
I removed the default values